src/rev_ai/models/insights/insights_job.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
 """Job model"""
 
-from ...asynchronous.job_status import JobStatus
+from ..asynchronous.job_status import JobStatus
 
 
-class TopicExtractionJob:
+class InsightsJob:
```

Comment:
I'm not sure how I feel about this change. They really are two different APIs that will evolve independently and the products are sold separately (further reinforcing the idea that they are separate).

It wouldn't be unreasonable to see how they could start to diverge in the future. Calling both Insights Jobs is a step in a different direction.

Reply:
Thats a fair point. I'll keep them separate